package.json

```diff
@@ -28,28 +28,31 @@
     "@typescript-eslint/eslint-plugin": "^3.0.0",
     "@typescript-eslint/parser": "^3.0.0",
     "autocannon": "^5.0.0",
-    "fastify": "^2.12.0",
+    "fastify": "^3.0.0",
     "graphql-tools": "^6.0.8",
     "pre-commit": "^1.2.2",
     "proxyquire": "^2.1.3",
     "snazzy": "^8.0.0",
     "standard": "^14.0.0",
     "tap": "^14.10.6",
-    "typescript": "^3.7.5"
+    "tsd": "^0.11.0",
+    "ws": "^7.2.1"
   },
   "dependencies": {
     "end-of-stream": "^1.4.4",
-    "fastify-plugin": "^1.6.0",
-    "fastify-static": "^2.6.0",
-    "fastify-websocket": "^1.0.0",
+    "fastify-plugin": "^2.0.0",
+    "fastify-static": "^3.0.0",
+    "fastify-websocket": "^2.0.0",
     "graphql": "^15.0.0",
     "graphql-jit": "^0.4.0",
     "http-errors": "^1.7.3",
     "mqemitter": "^4.0.0",
     "p-map": "^4.0.0",
     "readable-stream": "^3.5.0",
     "single-user-cache": "^0.3.0",
-    "tiny-lru": "^7.0.2",
-    "ws": "^7.2.1"
```

Comment:
why was `ws` moved to a devDependency?

Reply:
must be a bad merge on my part when resolving conflicts. fixed